RadarSDK/RadarAPIHelper.m

```diff
@@ -67,9 +67,39 @@ - (void)requestWithMethod:(NSString *)method
             }
 
             if (params) {
-                [req setHTTPBody:[NSJSONSerialization dataWithJSONObject:params options:0 error:NULL]];
+                NSNumber *prevUpdatedAtMsDiff = params[@"updatedAtMsDiff"];
```

Comment:
Now that we're collecting `locationMS` we could also pass up `updatedAt` (we probably already are) we could do this bookkeeping server side but I think let's do that in a v2 in the interest of getting this out.